presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -461,7 +461,7 @@ protected OperatorStats searchScanFilterAndProjectOperatorStats(QueryId queryId,
                 .stream()
                 .filter(summary -> nodeId.equals(summary.getPlanNodeId()) && summary.getOperatorType().equals("ScanFilterAndProjectOperator"))
                 .collect(MoreCollectors.onlyElement());
-        assertEventually(new Duration(5, SECONDS), operatorStatsSupplier::get);
+        assertEventually(new Duration(5, SECONDS), () -> assertThat(operatorStatsSupplier.get()).isNotNull());
```

Comment:
You can remove `assertEventually(new Duration(5, SECONDS), operatorStatsSupplier::get);` altogether

Reply:
done